pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -151,15 +160,18 @@ func (l *libcontainerAdapter) revertName(name string) CgroupName {
 	if l.cgroupManagerType != libcontainerSystemd {
 		return CgroupName(name)
 	}
+	return CgroupName(RevertFromSystemdToCgroupStyleName(name))
+}
 
+func RevertFromSystemdToCgroupStyleName(name string) string {
```

Comment:
the typical convention in past has been outside of the cm package, users always worked with `CgroupName` as it was the abstract name, and any attempt to compare it to something in the cgroupfs required going through the adapter and getting the concrete form.